Remotion/ObjectBinding/Web.ClientScript/Scripts/BocList.ts

```diff
@@ -503,14 +511,26 @@ class BocList
     {
       if (!PageUtility.Instance.IsInDom (scrollableContainer))
         return;
-  
+
+      BocList.FixValidationErrorOverflowSize(scrollableContainer);
       BocList.FixHeaderSize(scrollableContainer);
       BocList.FixHeaderPosition(tableContainer, scrollableContainer);
       setTimeout(resizeHandler, resizeInterval);
     };
     resizeHandler();
   }
 
+  private static FixValidationErrorOverflowSize(scrollableContainer: HTMLElement)
+  {
+    const width = LayoutUtility.GetWidth(scrollableContainer);
+
+    const overflowContainer: HTMLElement | null = scrollableContainer.querySelector('div.bocListTableBlock tr.bocListValidationRow div');
```

Comment:
Please use `Nullable<HTMLElement>` instead. Same type but we use the `Nullable<T>` helper everywhere else so we should use it here as well.